Makefile

```diff
@@ -30,14 +30,19 @@ CI_FLAGS =\
 .PHONY: dev_req
 dev_req: ## Installs dbt-* packages in develop mode along with only development dependencies.
 	@\
-	pip install -r dev-requirements.txt
-	pip install -r editable-requirements.txt
+	pip install -e ./core -r dev-requirements.txt
 
 .PHONY: dev
 dev: dev_req ## Installs dbt-* packages in develop mode along with development dependencies and pre-commit.
 	@\
 	pre-commit install
 
+.PHONY: dev-uninstall
```

Comment:
I just find this useful so that I don't need to blow away and remake my virtual environment. It's unrelated to this PR outside of the fact that I used it while troubleshooting.